timesketch/api/v1/resources.py

```diff
@@ -2148,6 +2158,84 @@ def get(self, searchindex_id):
         return self.to_json(searchindex)
 
 
+class UserListResource(ResourceMixin, Resource):
+    """Resource to get list of users."""
+
+    @login_required
+    def get(self):
+        """Handles GET request to the resource.
+
+        Returns:
+            List of usernames
+        """
+        return self.to_json(User.query.all())
+
+
+class GroupListResource(ResourceMixin, Resource):
+    """Resource to get list of groups."""
+
+    @login_required
+    def get(self):
+        """Handles GET request to the resource.
+
+        Returns:
+            List of group names
+        """
+        return self.to_json(Group.query.all())
+
+
+class CollaboratorResource(ResourceMixin, Resource):
+    """Resource to update sketch collaborators."""
+
+    @login_required
+    def post(self, sketch_id):
+        """Handles POST request to the resource.
+
+        Args:
+            sketch_id: Integer primary key for a sketch database model
+        """
+        sketch = Sketch.query.get_with_acl(sketch_id)
+        form = request.json
```

Comment:
why not use the forms and have the request.json call directly?

Reply:
As discussed offline - Trying to move away from the old forms library for validation. We pass json nowadays, and forms don't support list of strings for example.